garrysmod/lua/vgui/dform.lua

```diff
@@ -134,6 +134,61 @@ function PANEL:PropSelect( label, convar, models, height )
 
 end
 
+function PANEL:ToolPresets( group, cvarlist )

```

Comment:
Get rid of all changes in this file. We cannot put Sandbox exclusive code to DForm.

Reply:
Done in https://github.com/Facepunch/garrysmod/pull/1854/commits/b964f8796ea9c5184a31894da9ff2fef6b2a66f6